BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -34,6 +34,7 @@ public final class ConvivaAnalytics: NSObject {
     let playerHelper: BitmovinPlayerHelper
     // Workaround for player issue when onPlay is sent while player is stalled
     var isStalled: Bool = false
+    var isPlaybackStarted: Bool = false
```

Comment:
This needs to be reset for next playback. `internalInitializeSession` and `internalEndSession` can be good places to reset this to false.

Reply:
Removed this

Reply:
Reset this in internalEndSession